lib/clients/transports/call-transport.js

```diff
@@ -102,15 +101,15 @@ var handleHttpResponse = function handleHttpResponse(body, headers, client, apiC
   if (!jsonResponse.ok) {
     jsonError = new Error(jsonResponse.error);
     client.logger('error', 'Response not OK: ', jsonResponse.error);
-  }
```

Comment:
what if the scopes are relevant to the error that just occurred, and the client wanted this data in the `jsonResponse` object?